cluster/gce/addons/loadbalancing/cloud-provider-binding.yaml

```diff
@@ -3,6 +3,38 @@ kind: RoleBinding
 metadata:
   labels:
     addonmanager.kubernetes.io/mode: Reconcile
+  name: gce:cloud-provider
+  namespace: kube-system
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: gce:cloud-provider
+subjects:
+- kind: ServiceAccount
+  name: cloud-provider
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
```

Comment:
Techinically there's a slight risk to this, as it would clobber another role of the same name. I think it's _probably_ ok given the `gce:` prefix.

Reply:
It's not that likely that a cluster user has a role named `gce:cloud-provider`. Since we didn't get any reports reporting issue about roles named `cloud-provider` in `Reconcile` mode clobbering exist roles, I assume `gce:cloud-provider` is even safer than `cloud-provider` to have `Reconcile`.